### ISSUE TITLE
Option for redirecting landing page to docs/

### ISSUE THREAD

**Issue by capsulecorplab** [DocOps/asciidocsy-jekyll-theme#55]
Some documentation projects may not need a landing page, in which case it'd be useful to redirect directly to docs/

**Comment by briandominick** [comment]
You're definitely right, most sites won't need a landing page other than their main front page.

Generally, we want to be at `docs.companyname.com/` even, and then you'd use a page like the one sourced at `_docs/topic/intro.adoc` and give it a permalink of `/` to make it the index page.

**Comment by briandominick** [comment]
I'm going to resolve this once and for all for 0.2.1 patch release. I will make sure it is clear how to serve at different URLs.

**Comment by briandominick** [comment]
I think I had this matter mixed up with the `baseurl` issue we addressed in #73.

This theme is not determining where the docs are served in any implementation other than itself -- the docs for AsciiDocsy theme. This will be more apparent with the release of 0.3.0, as the theme will be a gem, and a separate repo for the AsciiDocsy docs will _use_ the gem as a plugin, just as all users will use the theme going forward. This will make designing your own URL structures much simpler.

Closing this issue for now.
